File: vbox-app/src/com/kedzie/vbox/api/IEventListener.java
package com.kedzie.vbox.api;

import java.io.IOException;

import com.kedzie.vbox.soap.KSOAP;

@KSOAP
public interface IEventListener extends IManagedObjectRef {

	public void handleEvent(@KSOAP("event") IEvent event) throws IOException;
	
}

File: vbox-app/src/main/java/com/kedzie/vbox/api/IEventListener.kt
package com.kedzie.vbox.api

import android.os.Parcelable
import com.kedzie.vbox.soap.Ksoap
import com.kedzie.vbox.soap.KsoapProxy

@KsoapProxy
@Ksoap
interface IEventListener : IManagedObjectRef, Parcelable {

    suspend fun handleEvent(event: IEvent)
}